pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -76,17 +97,28 @@ let
   withLuaJIT = !(stdenv.hostPlatform.isPower && stdenv.hostPlatform.is64bit) && !stdenv.hostPlatform.isRiscV;
 in rec { # un-indented
 
-inherit (common) cleanBrokenLinks;
 texliveYear = year;
 
 
 core = stdenv.mkDerivation rec {
   pname = "texlive-bin";
   inherit version;
 
-  inherit (common) src prePatch;
+  __structuredAttrs = true;
+
+  inherit (common) binToOutput src prePatch;
 
-  outputs = [ "out" "doc" "dev" ];
+  outputs = [ "out" "dev" "man" "info"
+    # all output packages, manually computed by listing $out/bin
+    # and matching each binary to a package according to binToOutput
+    "afm2pl" "aleph" "autosp" "axodraw2" "bibtex" "cjkutils" "ctie" "cweb"
+    "detex" "dtl" "dvi2tty" "dvicopy" "dvidvi" "dviljk" "dviout_util"
+    "dvipdfmx" "dvipos" "dvips" "fontware" "gregoriotex" "gsftopk" "hitex"
+    "kpathsea" "lacheck" "lcdftypetools" "m_tx" "makeindex" "mfware" "musixtnt"
+    "omegaware" "patgen" "pdftex" "pdftosrc" "pmx" "ps2eps" "ps2pk" "psutils"
+    "ptex" "seetexk" "synctex" "t1utils" "tex" "tex4ht" "texlive_scripts_extra"
+    "texware" "tie" "tpic2pdftex" "ttfutils" "uptex" "velthuis" "vlna" "web"
+    "xml2pmx" ];
 
   nativeBuildInputs = [
     pkg-config
```

Comment:
I probably broke cross-compiling here, we need `texlive.bin.core.web` and maybe also `cweb`, `tie`, `ctie`... some of those.

Reply:
Excellent, https://tug.org/texlive/doc/tlbuild.html#Cross-problems lists the binaries required for cross compiling:

> This approach would also be possible for the tools such as tangle used in the module texk/web2c to build the WEB programs, but that would require first building a native kpathsea library. To avoid this complication, cross compilation of programs written in (C)WEB requires sufficiently recent installed versions of tangle, ctangle, otangle, and tie.

Hence the outputs `core.web`, `core.cweb`, `core.omegaware`, `core.tie` should suffice, in theory. I don't have the means to test it though.

Reply:
added